src/main/java/hudson/plugins/plot/CSVSeries.java

```diff
@@ -267,6 +301,32 @@ private boolean excludePoint(String label, int index) {
         return retVal;
     }
 
+    /**
+     * Checks if the current label / header is known in the strExclusionSet (plain text or regex).
+     *
+     * @param label
+     * @return if the label is in the set
+     */
+    private boolean checkExclusionSet(String label) {
+        if (strExclusionSet.contains(label)) {
+            return true;
+        } else {
+            for (String s : strExclusionSet) {
+                try {
+                    if (label.matches(s)) {
```

Comment:
This broke the plugin for us since our excluded columns contain parentheses. Even though the items are not enclosed in quotes, they are interpreted as regular expressions leading to a `java.util.regex.PatternSyntaxException: Unclosed group near index 44`.

Reply:
@jochenberger can you please provide an example of test csv file and then the exclusion pattern you're using?

Reply:
@vgaidarji, I'm afraid I cannot share the data. I can however tell you that the exclusion pattern contains an opening parenthesis. Is that enough information to reproduce the issue?

Reply:
@jochenberger would you please also provide your used settings (like the InclusionFlag and the exclusionValues)?

Reply:
@dgetzlaf, you should be able to reproduce it with
```xml
<inclusionFlag>INCLUDE_BY_STRING</inclusionFlag>
<exclusionValues>x(y</exclusionValues>
```

Reply:
@jochenberger @dgetzlaf Was able to reproduce the issue in tests, opened a draft PR https://github.com/jenkinsci/plot-plugin/pull/57.

Reply:
Just to be clear, this is *not* about broken regexes, it's about literal string values (without quotes) being *Interpreted* as regexes.

Reply:
@jochenberger yeah I got that. 
Is the data below reflecting your specific case?
```
Avg,Median,90,min,max,samples,errors
515.33,196,1117,2,16550,97560,360
```

and `INCLUDE_BY_STRING` or `EXCLUDE_BY_STRING` = `"Avg", "(Median"`

Where `(Media` could be one of the CSV headers or not, depending on input CSV.
So you want to have a general plot configuration with `(` or `)` and have it interpreted as a simple String instead of trying to parse it as a regex. Correct?

Reply:
Almost. My exclusion values are not quoted so I assumed they would not be parsed as regexes anyway. At least that's how I understand the docs.

Reply:
@jochenberger should be fixed now by https://github.com/jenkinsci/plot-plugin/pull/57.

Reply:
I doubt that the fix is correct. What if `INCLUDE_BY_STRING` is `.` and data is 
```
x,.,y
1,2,3
```?
It should only include the second column because the value is not quoted (`"."`).